File: src/app/global/cli/CliException.java
package app.global.cli;

public class CliException extends Exception {
  public CliException(String message) {
    super(message);
  }

  public CliException(String message, Throwable cause) {
    super(message, cause);
  }

  public CliException(Throwable cause) {
    super(cause);
  }
}

File: kt/app/global/cli/CliException.kt
package app.global.cli

import java.lang.Exception

/**
 * @author Jack Meng
 */
class CliException : Exception {
    constructor(message: String?) : super(message) {}
    constructor(message: String?, cause: Throwable?) : super(message, cause) {}
    constructor(cause: Throwable?) : super(cause) {}
}